app/assets/javascripts/discourse/tests/helpers/site-settings.js

```diff
@@ -11,7 +11,6 @@ const CLIENT_SETTING_TEST_OVERRIDES = {
   enable_github_logins: true,
   authorized_extensions: "jpg|jpeg|png|gif|heic|heif|webp|svg|txt|ico|yml",
   anon_polling_interval: 30000,
-  flush_timings_secs: 5,
```

Comment:
I compared the front-end test CI runs before and after, and there seems to be no impact.